test/bounded-memory/mzcompose.py

```diff
@@ -236,25 +236,25 @@ class KafkaScenario(Scenario):
                     INSERT INTO t1 (f3) SELECT '{i}' || REPEAT('a', {PAD_LEN}) FROM generate_series(1, {REPEAT});
                     """
                 )
-                for i in range(0, ITERATIONS)
+                for i in range(0, ITERATIONS * 10)
             ]
         )
         + PgCdcScenario.MZ_SETUP
         + dedent(
             f"""
-            > SELECT * FROM v1; /* expect {ITERATIONS * REPEAT} */
-            {ITERATIONS * REPEAT}
+            > SELECT * FROM v1; /* expect {ITERATIONS * 10 * REPEAT} */
+            {ITERATIONS * 10 * REPEAT}
             """
         ),
         post_restart=dedent(
             f"""
             # We do not do DELETE post-restart, as it will cause OOM for clusterd
-            > SELECT * FROM v1; /* expect {ITERATIONS * REPEAT} */
-            {ITERATIONS * REPEAT}
+            > SELECT * FROM v1; /* expect {ITERATIONS * 10 * REPEAT} */
+            {ITERATIONS * 10 * REPEAT}
             """
         ),
         materialized_memory="4.5Gb",
-        clusterd_memory="3.5Gb",
+        clusterd_memory="1Gb",
```

Comment:
Had to both bump the #iterations and reduce the memory limit to make the original issue reproducible. @def- lmk if you'd rather have this as a separate test. But we should have _some_ regression test.

Reply:
Fine to have it in here I think.